service/controller/app/v1/resource/configmap/desired.go

```diff
@@ -55,6 +56,9 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      key.ChartConfigMapName(cr),
 			Namespace: r.chartNamespace,
+			Annotations: map[string]string{
+				annotation.Notes: "DO NOT EDIT",
```

Comment:
I feel like this should be more specific. I am not sure what should not be edited now. The annotation? Something else in there? The whole CR? And then, why though?

Reply:
Yes I see what you mean. Although I got feedback from @puja108 to make the message less specific. https://github.com/giantswarm/giantswarm/issues/8863#issuecomment-589657132

The reason is because this configmap is generated by app-operator and consumed by chart-operator. So the user should edit the user configmap for the app CR.

What about changing it to  `giantswarm.io/notes: DO NOT EDIT. Values managed by app-operator.`?

Reply:
I am not big about the words in particular. I just think it should transport a certain message without raising more questions. /cc @giantswarm/sig-ux

Reply:
Got feedback from SIG UX and extended the message to have more context.

@xh3b4sd PTAL